streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -91,11 +98,17 @@ public void process(Exchange exchange) {
 			log.trace("Skipping disabled events ..." + table);
 			return;
 		}
-		if (payload.get("uuid") == null) {
-			log.error(String.format("No uuid column for table %s ignoring payload %s ", table, payload));
-			return;
+		
+		String uuid = "";
+		
+		if (payload.get("uuid") != null) {
+			uuid = payload.get("uuid").toString();
+		} else {
+			getUuidUtil = new GetUuidUtil(jdbcConnectionUtil);
+			uuid = getUuidUtil.getUuid(config.getParentTable(), config.getParentForeignKey(), config.getChildPrimaryKey(),
```

Comment:
Should we account for `null` values here? I mean in case the configuration file is mis-configured and doesn't have this field for all tables that don't have `uuid` column.

Reply:
@bashir2 am not really understanding what you mean by "`mis-configured and doesn't have this field for all tables that don't have uuid column`."
you mean in the case where the child table doesnt have the reference to the parent foreign column?

Reply:
Think about a mis-configured JSON configuration, e.g., these is no `parentTable` in `patient` record of `dbz_event_to_fhir_config.json`.

Reply:
Did you make any changes for this comment, I can't find any?